tests/Type/data/model-factories.php

```diff
@@ -8,6 +8,7 @@
 assertType('Database\Factories\UserFactory', User::factory());
 assertType('Database\Factories\UserFactory', User::factory()->new());
 assertType('App\User', User::factory()->createOne());
+assertType('App\User', User::factory()->createOneQuietly());
```

Comment:
Instead of adding this here, please add similar tests after every `createQuietly`

Reply:
Can you help me understand why you requested this change?
My thinking was that createOneQuietly is more like createOne than createQuietly.

Reply:
Yes, it is more like `createOne`. I just wanted you to add more tests. Which you can do it by adding them just below of every `createQuietly`

Like `assertType('Illuminate\Database\Eloquent\Collection<int, App\User>', User::factory(2)->createOneQuietly());` and `assertType('App\User', User::factory()->createOneQuietly());`

Reply:
```
 assertType('Illuminate\Database\Eloquent\Collection<int, App\User>', User::factory(2)->createOneQuietly());
```
Are you sure about this? I think this is broken usage, why would you tell the factory to create two items and then ask it for one?
The latter of the two is already there, if do you want to duplicate it?

Reply:
While it doesn't make sense, it is a valid usage. It's not broken. We would test that `createOneQuietly` still returns single model even after calling `User::factory(2)`

The two I wrote above are just examples. I want you to add new test cases based on the previous test cases. If you read the file you'll notice a pattern. So I just want you to add `createOneQuietly` tests like the rest of them/

Reply:
I'm not sure which expectations we need to verify in these additional tests. The createQuietly method has two return types, Model and Collection<int,Model>, I see how you would want to test both. createOneQuietly only has one return type. 
I also see that there is only one test for createOne, possibly for the same reason. 
I hope I'm not annoying with all my questions, but I'm trying to understand the why.

Reply:
It is just additional safety. While unlikely, I don't want to break something in the future accidentally. So having these tests would give us a confidence and doesn't cost anything 😄

Reply:
I added the tests. I saw that lines 33-37 are equal to lines 38-41. Is this intentional?

Reply:
No, it's not intentional. Must be mistake. You can delete one of them.

Reply:
I deleted them.